### ISSUE TITLE
Cant get on friends server

### ISSUE THREAD

**Issue by APeng121** [FTBTeam/FTB-Revelation-2#205]
*FTB Revelation on Twitch mod launcher, ver. 2.7.0

Getting onto the modpack takes a long time but thats just my bad computer however i cannot join a friends server. I am able to join a public server and single player server just fine however something is blocking me from joining my friends server and im not sure why or how to fix it.
Friends have told me they've seen me get into the server and leave right away although I never see the world and usually just see a blank dirt screen..

https://paste.feed-the-beast.com/view/c38120aa

![minecraftftb](https://user-images.githubusercontent.com/48573615/54401554-8d3ef000-469e-11e9-93c4-46d389469ca7.PNG)
 to paste the text of your log/crash file -->

*So basically I just cant get onto my friends server but can get on public ones and i have no idea on anything computer so im sorry if im difficult to work with but any help is appreciated!

**Comment by FiveEYZ** [comment]
my sugestion is to tell ur friend to add the mod [RandomPatches](https://minecraft.curseforge.com/projects/randompatches) to the server (not needed on the client) it has lots of tweaks and fixes and should fix the login issue, cuz it did for me.
